runhouse/rns/rns_client.py

```diff
@@ -659,3 +671,25 @@ def contents(self, name_or_path, full_paths):
         return folder(name=name_or_path, path=folder_url).resources(
             full_paths=full_paths
         )
+
+    def send_status(self, status: ResourceStatusData, cluster_rns_address: str):
```

Comment:
We need to catch all other errors so any failures here don't nuke the loop or cluster servlet.

Reply:
Should this be async?

Reply:
Guess async doesn't rly matter because its in its own thread, but can for consistency

Reply:
> We need to catch all other errors so any failures here don't nuke the loop or cluster servlet.

The cluster servlet has everything in a try: except: so it'll catch it